helm/alfresco-content-services/values.yaml

```diff
@@ -122,7 +122,9 @@ repository:
 apiexplorer:
   ingress:
     path: /api-explorer
-ooiService:
+alfresco-connector-ms365:
+  enabled: false
```

Comment:
So you decided to move th toggler under the "configurator" node `alfresco-connector-ms365`?
I like the idea but there may be impact doing that (not sure backward compat will be that easy to ensure)